internal/gqltestutil/permissions.go

```diff
@@ -162,6 +162,23 @@ query BitbucketProjectPermissionJobs($projectKeys: [String!], $status: String, $
 	}
 }
 
+func (c *Client) AuthzProviders() ([]string, error) {
+	const query = `
+query {
+	authzProviders
+}`
```

Comment:
Nitpick, but I do think multi-line strings with funny intendation are uglier than a single line:

```suggestion
	const query = `query { authzProviders }`
```